packages/components/src/tooltip/test/index.js

```diff
@@ -11,123 +12,124 @@ import Tooltip from '../';
+
+			render(
 				<Tooltip text="Help text">
-					<button
-						onMouseEnter={ originalFocus }
-						onFocus={ originalFocus }
-					>
-						Hover Me!
-					</button>
+					<button>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-			button.simulate( 'focus', event );
+			const button = screen.getByRole( 'button' );
+			await user.hover( button );
+
+			expect(
+				screen.getByRole( 'button', { name: 'Hover Me!' } )
+			).toBeInTheDocument();
 
-			const popover = wrapper.find( 'ForwardRef(Popover)' );
-			expect( originalFocus ).toHaveBeenCalledWith( event );
-			expect( popover ).toHaveLength( 1 );
+			setTimeout( () => {
+				expect( screen.getByText( 'Help text' ) ).toBeInTheDocument();
+				jest.runOnlyPendingTimers();
+				jest.useRealTimers();
+			}, TOOLTIP_DELAY );
 		} );
 
-		it( 'should show not popover on focus as result of mousedown', async () => {
-			const originalOnMouseDown = jest.fn();
-			const originalOnMouseUp = jest.fn();
-			const wrapper = mount(
+		it( 'should show popover on focus', () => {
+			const originalFocus = jest.fn();
+			render(
 				<Tooltip text="Help text">
-					<button
-						onMouseDown={ originalOnMouseDown }
-						onMouseUp={ originalOnMouseUp }
-					>
-						Hover Me!
-					</button>
+					<button onFocus={ originalFocus }>Hover Me!</button>
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-
-			let event;
-
-			event = { type: 'mousedown' };
-			button.simulate( event.type, event );
-			expect( originalOnMouseDown ).toHaveBeenCalledWith(
+			const button = screen.getByRole( 'button' );
+			button.focus();
+			expect( originalFocus ).toHaveBeenCalledWith(
 				expect.objectContaining( {
-					type: event.type,
+					type: 'focus',
 				} )
 			);
+			expect( screen.getByText( 'Help text' ) ).toBeInTheDocument();
+		} );
 
-			event = { type: 'focus', currentTarget: {} };
-			button.simulate( event.type, event );
-
-			const popover = wrapper.find( 'Popover' );
-			expect( popover ).toHaveLength( 0 );
+		it( 'should not show popover on focus as result of mouse click', async () => {
+			const user = userEvent.setup( {
+				advanceTimers: jest.advanceTimersByTime,
+			} );
+			jest.useFakeTimers();
 
-			event = new window.MouseEvent( 'mouseup' );
-			await act( async () => document.dispatchEvent( event ) );
-			expect( originalOnMouseUp ).toHaveBeenCalledWith(
-				expect.objectContaining( {
-					type: event.type,
-				} )
+			render(
+				<Tooltip text="Help text">
+					<button>Hover Me!</button>
+				</Tooltip>
 			);
+
+			const button = screen.getByRole( 'button' );
+			await user.click( button );
+			setTimeout( () => {
+				expect(
+					screen.queryByText( 'Help text' )
+				).not.toBeInTheDocument();
+				jest.runOnlyPendingTimers();
+				jest.useRealTimers();
+			}, TOOLTIP_DELAY );
 		} );
 
-		it( 'should show popover on delayed mouseenter', () => {
+		it( 'should show popover on delayed mouseenter', async () => {
```

Comment:
I believe that we can delete this test, since the previous one already checks for the tooltip to appear on hover after a given delay

Reply:
Fixed by c8b1c0a